### DISCUSSION TITLE
Is it possible to skip skipping settings/portable on install

### DISCUSSION THREAD

**Discussion by sKGu8gxV** [ScoopInstaller/Scoop#4733] [Help and Questions] [answered]
I'm wondering if there's a way to install apps without getting the "Make Portable" or "Use these settings". Please let me clarify. If you install for example Firefox these directories are linked: distribution and profile meaning if I were to use these as is, they would change the settings I have in my system already. 

In Notepad++ the following links are created: backup, cloud, plugins, themes, userDefineLangs. 

I would prefer to be able to not get these linked/installed on installation. Is this possible?

Err sorry for making this a comment instead of the actual thread, I'm not used to Git.

**Comment by rashil2000** [comment, marked as answer]
What do you mean? Do you have an example?

**Reply by sKGu8gxV** [reply]
Sorry not used to the UI here and got distracted. I hope it's clearer now.

**Reply by rashil2000** [reply]
No worries. I understand your question now.

No, it's not possible to do this as of now. You will need to create your own manifest without the persist fields if you want to have the non-portable mode.

Install options for apps has been requested before, it is being tracked here - https://github.com/ScoopInstaller/Scoop/issues/3736

**Reply by sKGu8gxV** [reply]
Thanks for your hard work and your reply!
